tests/suites/test_suite_psa_crypto.data

```diff
@@ -6118,7 +6118,7 @@ depends_on:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PRF
 derive_key_exercise:PSA_ALG_TLS12_PRF(PSA_ALG_SHA_256):"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":"000102030405060708090a0b0c":"f0f1f2f3f4f5f6f7f8f9":PSA_KEY_TYPE_DERIVE:400:PSA_KEY_USAGE_DERIVE:PSA_ALG_HKDF(PSA_ALG_SHA_256)
 
 PSA key derivation: HKDF-SHA-256 -> ECC secp256r1, exercise ECDSA
-depends_on:PSA_WANT_ALG_ECDSA:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256
+depends_on:PSA_WANT_ALG_ECDSA:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_ECC_KEY_PAIR:PSA_WANT_ECC_SECP_R1_256:MBEDTLS_ECP_DP_SECP256R1
```

Comment:
Minor: it's equivalent in practice, but as a matter of principle I think this should be `MBEDTLS_PSA_BUILTIN_ECC_SECP_R1_256`. In the long run we want to completely eliminate `MBEDTLS_` macros from the PSA space, other than perhaps `MBEDTLS_PSA_` macros.

Again, in this case the two names are equivalent, and the dependency is correct (until we implemented driver support for ECC key derivation) but using the `MBEDTLS_PSA_` macro makes future automated checking easier, when we work on eliminating `MBEDTLS_` dependencies that, unlike that one, are not correct.

Reply:
I didn't think about this detail, but I totally agree! :)